deploy-addresses/soneium.json

```diff
@@ -0,0 +1,3 @@
+{
+  "UniversalRouterV2": "0x4cded7edf52c8aa5259a54ec6a3ce7c6d2a455df"
```

Comment:
UnsupportedProtocol: 0x8702463e73f74d0b6765aBceb314Ef07aCb92650

Reply:
i wonder if we should name this one something special because it doesnt support v2 or v3 😓

Reply:
yeah i wasn't sure how naming should go like for v1.2, if the universal router supported v2 right away, do we still call it v1_2_v2support? or just call it v1_2?